scripts/tests/test_validate_docstrings.py

```diff
@@ -1,8 +1,10 @@
-import os
-import sys
-
-import numpy as np
 import pytest
+import string
+import random
+import numpy as np
+
+validate_docstrings = pytest.importorskip("validate_docstrings")
```

Comment:
I guess I'm missing something obvious, but why is this needed? can't we simply `import validate_docstrings`?

Reply:
That's vestigial paranoia from when we skipped tests if we couldn't see `validate_docstrings`.  I agree that we can just import this without the protection.